layouts/_default/baseof.html

```diff
@@ -15,7 +15,11 @@
 	{{ printf `<link rel="%s" type="%s" href="%s" title="%s">` .Rel .MediaType.Type .RelPermalink $.Site.Title | safeHTML }}
 	{{ end -}}
 	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,700">
-	<link rel="stylesheet" href="{{ "css/style.css" | relURL }}">
+
+	{{ $sassTemplate := resources.Get "css/style.css" }}
+	{{ $style := $sassTemplate | resources.ExecuteAsTemplate "css/style.css" . }}
```

Comment:
```
{{ $style := resources.Get "css/style.css" | resources.ExecuteAsTemplate "css/style.css" . -}}
<link rel="stylesheet" href="{{ $style.RelPermalink }}">
```

As long as we have a simple Hugo pipe chain, there is not much point in splitting it into two lines.